.github/workflows/ci_check_translations.yml

```diff
@@ -0,0 +1,43 @@
+name: check translations
+
+on:
+  push:
+    branches:
+      - master
+      - development
+      - release
+  pull_request:
+
+jobs:
+  check_translations:
+    runs-on: ubuntu-18.04
+    services:
```

Comment:
Do we need the services for this? We are just running static code analysis.

Reply:
It is connecting to the DB to check if the values are in the context

Reply:
Of course, got it.